orca-web/src/test/groovy/com/netflix/spinnaker/orca/controllers/TaskControllerSpec.groovy

```diff
@@ -140,36 +150,39 @@ class TaskControllerSpec extends Specification {
 
   void '/applications/{application}/pipelines should only return pipelines from the past two weeks, newest first'() {
     given:
-    def now = new Date()
     def pipelines = [
-      [pipelineConfigId: "1", startTime: (new Date() - daysOfExecutionHistory).time - 1, id: 'old'],
-      [pipelineConfigId: "1", startTime: (new Date() - daysOfExecutionHistory).time + 1, id: 'newer'],
+      [pipelineConfigId: "1", startTime: clock.instant().minus(daysOfExecutionHistory, DAYS).minusMillis(1).toEpochMilli(), id: 'old'],
+      [pipelineConfigId: "1", startTime: clock.instant().minus(daysOfExecutionHistory, DAYS).plusMillis(1).toEpochMilli(), id: 'newer'],
```

Comment:
This was the cause of the flakiness — multiple `new Date` calls in assertions that rely on millisecond precision.